dist/tools/pyterm/pyterm

```diff
@@ -267,6 +267,14 @@ class SerCmd(cmd.Cmd):
                 return "PYTERM_" + line[1:]
         return line
 
+    def _prompt_exit(self):
+        reply = ''
+        while reply not in ('y', 'n'):
+            reply = input('Exit pyterm ([y]/n)? ').lower()
```

Comment:
Maybe put a `\n` before this?

I have

```
> Exit pyterm ([y]/n)?
```

while with iPython I get

```
In [1]: 
Do you really want to exit ([y]/n)?
```

;-)

Reply:
Why not!
I fixed a crash of the script by catching a ctrl+d while prompting the user. IPython closes in this case.

Reply:
Done, directly squashed